src/standard/gestures.html

```diff
@@ -41,12 +41,24 @@
   // Check for touch-only devices
   var IS_TOUCH_ONLY = navigator.userAgent.match(/iP(?:[oa]d|hone)|Android/);
 
+  // Check for sourceCapabilities, used to distinguish synthetic events
+  var HAS_SOURCE_CAPS = 'sourceCapabilities' in UIEvent.prototype;
+
   // touch will make synthetic mouse events
   // `preventDefault` on touchend will cancel them,
   // but this breaks `<input>` focus and link clicks
   // disable mouse handlers for MOUSE_TIMEOUT ms after
   // a touchend to ignore synthetic mouse events
   var mouseCanceller = function(mouseEvent) {
+    if (HAS_SOURCE_CAPS) {
```

Comment:
Is this `if` only to save the `if` on line 58? You already test for existence of `sourceCapabilities` on that line, so seems like extra code for no gain.